mlflow/azureml/__init__.py

```diff
@@ -248,7 +248,8 @@ def _create_dockerfile(output_path, mlflow_path=None):
                         Dockerfile command for MLflow installation will install MLflow from this
                         directory. Otherwise, it will install MLflow from pip.
     """
-    docker_cmds = ["RUN pip install azureml-sdk"]
+    docker_cmds = ["RUN apt-get update && apt-get install default-jre"]
```

Comment:
```suggestion
    docker_cmds = ["RUN apt-get update && apt-get install -y default-jre"]
```

Reply:
Without the `-y` flag, installation aborts due to a lack of affirmative installation input:
```
Do you want to continue? [Y/n] Abort.
The command '/bin/sh -c apt-get update && apt-get install default-jre' returned a non-zero code: 1
2019/08/22 19:54:46 Container failed during run: acb_step_0. No retries remaining.
failed to run step ID: acb_step_0: exit status 1
```